tests/tests/health.rs

```diff
@@ -57,9 +64,16 @@ async fn can_restart_node_with_transactions() {
 
     {
         // Given
-        let database =
-            CombinedDatabase::open(tmp_dir.path(), capacity, Default::default(), 512)
-                .unwrap();
+        let database = CombinedDatabase::open(
+            tmp_dir.path(),
+            Default::default(),
+            DatabaseConfig {
+                cache_capacity: Some(capacity),
+                max_fds: 512,
+                columns_policy: Default::default(),
+            },
```

Comment:
```suggestion
            DatabaseConfig::config_for_tests(),
```

it is okay to disable cache here as well